pkg/collateral/control.go

```diff
@@ -46,6 +46,9 @@ type Control struct {
 	// EmitBashCompletion controls whether to produce bash completion files.
 	EmitBashCompletion bool
 
+	// EmitZshCompletion controls whether to produce bash completion files.
```

Comment:
Comment should be "zsh completion files"?

Reply:
Done.